src/lib/addons/datadog-definition.ts

```diff
@@ -62,6 +62,21 @@ const dataDogDefinition: IAddonDefinition = {
             sensitive: true,
             type: 'textfield',
         },
+        {
+            name: 'bodyTemplate',
+            displayName: 'Body template',
+            placeholder: `{
+"event": "{{event.type}}",
+"createdBy": "{{event.createdBy}}",
+"featureToggle": "{{event.data.name}}",
+"timestamp": "{{event.data.createdAt}}"
```

Comment:
You can add 2 spaces for each line here to display the value with indentation, like in `webhook-definition`.